contents/blog/spaceborne-construction-detection-and-progress-tracking.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
the headings need to be changed, see [here](https://github.com/novitai/novitai.github.io/tree/master/contents/blog)

`subtitle` needs to be `description`
`layout` and `background` are unused now